pkg/kubectl/cmd/auth/cani.go

```diff
@@ -19,34 +19,45 @@ package auth
 import (
 	"errors"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
+	"sort"
 	"strings"
 
 	"github.com/spf13/cobra"
 
 	authorizationv1 "k8s.io/api/authorization/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	authorizationv1client "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	rbacv1helpers "k8s.io/kubernetes/pkg/apis/rbac/v1"
```

Comment:
This will break the work we're currently undergoing to split kubectl out of core repo. You should not use anything from internal k8s.

Reply:
@seans3 fyi